File: Data Structures & Algorithms/max-water-container/submission-0.cpp
class Solution {
public:
    int maxArea(vector<int>& nums) {
        int n = nums.size();
        int l = 0;
        int r = n-1;
        int maxWater = 0;
        while(l<r)
        {
            int wd = r-l;
            int ht = min(nums[r],nums[l]);
            maxWater = max(maxWater,wd*ht);
            if(nums[r]<nums[l]) r--;
            else l++;
        }
        return maxWater;
    }
};

File: Data Structures & Algorithms/remove-element/submission-0.py
class Solution:
    def removeElement(self, nums: List[int], val: int) -> int:
        n = len(nums)
        i = j = 0
        while i<n:
            if nums[i]!=val:
                nums[i],nums[j] = nums[j],nums[i]
                j+=1
            i+=1
        return j